cpp/src/randomforest/randomforest.cu

```diff
@@ -326,6 +326,17 @@ void build_treelite_forest(ModelHandle* model,
   }
 }
 
+ModelHandle tl_mod_handle(ModelHandle* model,
+                          std::vector<unsigned char>& data) {
+  std::cout << "inside the C++ code" << std::flush << std::endl;
+  std::cout << data.size() << std::flush << std::endl;
+  const char* filename = std::tmpnam(nullptr);
+  std::ofstream file(filename, std::ios::binary);
+  file.write((char*)&data[0], data.size());
+  TREELITE_CHECK(TreeliteLoadProtobufModel(filename, model));
```

Comment:
Why can't we use `build_treelite_forest` on the unpickled random forest? Seems like that uses the exact same code in the case where we have protobuf bytes already.

Reply:
We can, i kept them seperate as it was easier for me to debug and the fuction without messing with the `build_treelite_forest` function